diff --git a/node_modules/ytdl-core/lib/utils.js b/node_modules/ytdl-core/lib/utils.js
index 705c8dd..15c7173 100644
--- a/node_modules/ytdl-core/lib/utils.js
+++ b/node_modules/ytdl-core/lib/utils.js
@@ -60,7 +60,7 @@ const ESCAPING_SEQUENZES = [
   { start: "'", end: "'" },
   { start: '`', end: '`' },
   // RegeEx
-  { start: '/', end: '/', startPrefix: /(^|[[{:;,])\s?$/ },
+  { start: '/', end: '/', startPrefix: /(^|[[{:;,/])\s?$/},
 ];
 
 /**
@@ -97,9 +97,17 @@ exports.cutAfterJS = mixedJson => {
   // Go through all characters from the start
   for (i = 0; i < mixedJson.length; i++) {
     // End of current escaped object
-    if (!isEscaped && isEscapedObject !== null && mixedJson[i] === isEscapedObject.end) {
-      isEscapedObject = null;
-      continue;
+    for (i = 0; i < mixedJson.length; i++) {
+      // End of current escaped object
+      if (!isEscaped && isEscapedObject !== null && mixedJson[i] === isEscapedObject.end) {
+        // Ignore character inside RegEx character set with a range of 30 characters
+        let charsetStart = mixedJson.substring(i - 15, i).search(/\[[^\]]*/)
+        let charsetEnd = mixedJson.substring(i, i + 15).search(/[^\[]*\]/)
+        if (mixedJson[i] == '/' && charsetStart !== -1 && charsetEnd !== -1)
+          continue;
+  
+        isEscapedObject = null;
+        continue;
     // Might be the start of a new escaped object
     } else if (!isEscaped && isEscapedObject === null) {
       for (const escaped of ESCAPING_SEQUENZES) {
